storage/templates_test.go

```diff
@@ -77,4 +77,16 @@ func TestTemplate(t *testing.T) {
 	err = store.UpdateTemplate(&templates[1])
 	assert.Nil(t, err)
 
+	p := NewPaginationCursor("/api/templates", 10)
```

Comment:
We'll need to test the pagination here as well, moving forwards and backward, look at the other tests for this.